Packaging.props

```diff
@@ -4,6 +4,7 @@
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
     <PackageLicenseFile>$(ProjectDir)pkg/dotnet_library_license.txt</PackageLicenseFile>
     <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
+    <SyncInfoFile Condition="'$(SyncInfoFile)' == ''">$(ObjDir)version.txt</SyncInfoFile>
```

Comment:
Yes, it's just providing a place to override it, but it's not necessary.  I can remove it.